vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -175,12 +182,16 @@ fire_and_forget WinRTWebSocketResource::PerformPing() noexcept {
     } else {
       if (self->m_errorHandler) {
         auto error = winrt::hresult_error(result, winrt::hresult_error::from_abi);
-        self->m_errorHandler({winrt::to_string(error.message()), ErrorType::Ping});
+        std::stringstream hexCode;
+        hexCode << "[0x" << std::hex << error.code() << "] ";
+        self->m_errorHandler({hexCode.str() + winrt::to_string(error.message()), ErrorType::Ping});
       }
     }
   } catch (hresult_error const &e) {
     if (self->m_errorHandler) {
-      self->m_errorHandler({winrt::to_string(e.message()), ErrorType::Ping});
+      std::stringstream hexCode;
+      hexCode << "[0x" << std::hex << e.code() << "] ";
```

Comment:
>hexCode << "[0x" [](start = 6, length = 16)

Please create a small function that returns this result, something like:
string ErrorCodeToString(e.code())

Reply:
Done